Gemfile

```diff
@@ -1,5 +1,7 @@
 source "https://rubygems.org"
 
+gem "refinerycms-authentication-devise", '~> 1.0.4'
```

Comment:
I know why you put this, but this is a bit dangerous, because it makes this extension rely on this implementation _implicitly_.

Just noting this down.